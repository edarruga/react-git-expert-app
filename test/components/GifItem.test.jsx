import { render } from "@testing-library/react";
import {screen } from "@testing-library/react";
import { GifItem } from "../../src/components/GifItem";

describe('Pruebas de GifItem', () => {

    const title='Link';
    const url='http://zeldaTerasOfTheKingdom/link.jpg';

    test('debe de hacer mach con el snapshot', () => {
        
        const {container}=render(<GifItem title={title} url={url} />);
        expect(container).toMatchSnapshot();
    });

    test('',()=>{
        render(<GifItem title={title} url={url} />);
        expect(screen.getByRole('img').src).toBe(url.toLowerCase());
    });
});