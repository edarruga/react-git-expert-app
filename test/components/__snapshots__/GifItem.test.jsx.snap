// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas de GifItem debe de hacer mach con el snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="Link"
      src="http://zeldaTerasOfTheKingdom/link.jpg"
    />
    <p>
      Link
    </p>
  </div>
</div>
`;
